syntax = "proto3";
import "google/protobuf/timestamp.proto";
package models.db;


message Season{
    string _id = 1; 
    string season_id = 2; 
    string league_id = 3; 
    google.protobuf.Timestamp season_start_date = 4; 
    google.protobuf.Timestamp season_end_date= 5; 

}